token-swap/program/fuzz/fuzz_targets/instructions.rs

```diff
@@ -0,0 +1,258 @@
+#![no_main]
+
+use spl_token_swap_fuzz::{get_token_balance, AccountData, TokenSwapAccountInfo};
+
+use spl_token_swap::{
+    curve::{
+        base::{CurveType, SwapCurve},
+        constant_product::ConstantProductCurve,
+    },
+    error::SwapError,
+    instruction::{Deposit, Swap, Withdraw},
+};
+
+use spl_token::error::TokenError;
+
+use arbitrary::Arbitrary;
+use libfuzzer_sys::fuzz_target;
+use std::collections::{HashMap, HashSet};
```

Comment:
Other than these dependencies, which can be removed if desired, I think this could actually run on bpf as well, so we can make a `fuzz-bpf`command too!